### PR TITLE
Move MsGraph custom action to netstandard2.0 by not using ToHashSet()

### PR DESCRIPTION
**Pull request by thomas-chung**
### Purpose
*What is the context of this pull request? Why is it being done?*

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [x] I have commented my code, particularly in hard-to-understand areas	
- [x] I have added or updated the appropriate tests	
- [x] I have updated related documentation

### CODE REVIEW

**Review by lauren-mills — APPROVED** [review]

File: experimental/components/MsGraph/Actions/GetContacts.cs [hunk 1]
```diff
@@ -90,7 +90,10 @@ protected override async Task<List<CalendarSkillContactModel>> CallGraphServiceW
 
             if (people?.Count > 0)
             {
-                var existingResult = contactsResult.SelectMany(c => c.EmailAddresses).ToHashSet();
+                var result = contactsResult.SelectMany(c => c.EmailAddresses);
```
Comment (peterinnesmsft) [inline experimental/components/MsGraph/Actions/GetContacts.cs:93]: I would just move the IEnumerable straight into the constructor.

File: experimental/components/MsGraph/Actions/GetContacts.cs [hunk 2]
```diff
@@ -90,7 +90,10 @@ protected override async Task<List<CalendarSkillContactModel>> CallGraphServiceW
...
+
+                var existingResult = new HashSet<string>(result);
```
Comment (peterinnesmsft) [inline experimental/components/MsGraph/Actions/GetContacts.cs:95]: Presumably this should use StringComparer.OrdinalIgnoreCase?

**Review by peterinnesmsft — CHANGES_REQUESTED** [review]

File: experimental/components/MsGraph/Actions/GetContacts.cs (hunk 2 above)
Comment (thomas-chung) [reply experimental/components/MsGraph/Actions/GetContacts.cs:95]: The default behavior from before does not use ordinal ignore case. We might as well fix it.

File: experimental/components/MsGraph/Actions/GetContacts.cs (hunk 1 above)
Comment (thomas-chung) [reply experimental/components/MsGraph/Actions/GetContacts.cs:93]: Sure!

**Review by lauren-mills — APPROVED** [review]

**Pull request merged**